### DISCUSSION TITLE
chmod of auto backup script not permitted after #857

### DISCUSSION THREAD

**Discussion by planetxpress** [mbround18/valheim-docker#865] [Help!] [answered]
All of a sudden my installation that has been unchanged for quite a long time is no longer able to start up.

The container is crashing at start up with the following error.
```
valheim[2752628]: chmod: changing permissions of '/home/steam/valheim/cron.d/auto-backup': Operation not permitted
```
If I disable auto-backups, the container is able to successfully start.

I believe this may be related to changing the location of the cron auto backup file at https://github.com/mbround18/valheim-docker/pull/857/commits/153bff66683891e294951bf28bec00d0b8401485#diff-87de5bbe5413b2e057cf468061315358b8ef3a958d2eca9fad4db0bcfcf1288fL108-R119

Moving to `$HOME/valheim/cron.d` now seems to have put the cron file on the suggested bind mount `./valheim/server:/home/steam/valheim` whereas before it remained within the container.

What is somewhat confusing to me is that the files on the bind mount are all chown'ed to 1000:1000, and the container runs with `-e PUID=1000 -e PGID=1000`. But inspecting the steam user in the container, that doesn't seem to be the case as the steam user is running as uid 111
```
docker exec -it valheim id steam
uid=111(steam) gid=1000(steam) groups=1000(steam)
```
So I am curious what the suggested config would be to get auto-backups working again. Thanks again for all your efforts maintaining this container.

**Comment by jhigginson** [comment]
I'm having the same issue but with setting the cron for auto-updates.

**Reply by mbround18** [reply]
well thats not good, lets move that folder back

**Reply by mbround18** [reply, marked as answer]
@jhigginson and @planetxpress  try  `mbround18/valheim:sha-5aae22e`

https://github.com/mbround18/valheim-docker/pull/867

**Reply by planetxpress** [reply]
Works!

**Reply by jhigginson** [reply]
Works for me as well.

**Reply by mbround18** [reply]
@planetxpress and @jhigginson 
Fix is in https://github.com/mbround18/valheim-docker/releases/tag/v3.0.2 :)
